src/conn/conn_layered.c

```diff
@@ -190,6 +190,19 @@ __disagg_pick_up_checkpoint(WT_SESSION_IMPL *session, uint64_t meta_lsn, uint64_
             md_cursor->set_value(md_cursor, cfg_ret);
             WT_ERR(md_cursor->insert(md_cursor));
             __wt_free(session, cfg_ret);
+
+            /*
+             * If there is a matching data handle, it out of date, as the metadata has changed. Mark
+             * it so, the data handle caches will know to ignore it, and it will eventually age out.
```

Comment:
Nifty - this is a nice spot to flag that.

I think you are missing a word in `it out of date`

Reply:
fixed, thanks